src/main/java/uk/gov/justice/hmpps/prison/service/SearchOffenderService.java

```diff
@@ -51,7 +47,7 @@ public Page<OffenderBooking> findOffenders(final SearchOffenderRequest request)
         Objects.requireNonNull(request.getLocationPrefix(), "locationPrefix is a required parameter");
         log.info("Searching for offenders, criteria: {}", request);
 
-        final Set<String> caseloads = securityUtils.isOverrideRole("SYSTEM_USER") ? Set.of() : userService.getCaseLoadIds(request.getUsername());
+        final Set<String> caseloads = userService.getCaseLoadIds(request.getUsername());
```

Comment:
SYSTEM_USER check here is redundant as Resource level check enforces a different role